docs/resources/cdn_domain.md

```diff
@@ -745,6 +760,38 @@ The `ip_filter` block support:
   by commas (,). IPv6 addresses are supported. Duplicate IP addresses and IP address segments will be removed.
   Addresses with wildcard characters are not supported, for example, `192.168.0.*`.
 
+<a name="origin_request_url_rewrite_object"></a>
+The `origin_request_url_rewrite` block support:
+
+* `priority` - (Required, Int) Specifies the priority of a URL rewrite rule. The priority of a rule is mandatory and
+  must be unique. The rule with the highest priority will be used for matching first. The value ranges from **1** to
+  **100**.
```

Comment:
It is best to explain the relationship between number size and priority

Reply:
OK